pkg/daemon/daemon.go

```diff
@@ -531,39 +581,18 @@ func (p *ptpProcess) cmdRun(stdoutToSocket bool) {
 				processStatus(&c, p.name, p.messageTag, PtpProcessUp)
 				for scanner.Scan() {
 					output := scanner.Text()
+					if p.pmcCheck {
+						p.pmcCheck = false
+						go p.updateClockClass(&c)
+					}
+
 					if regexErr != nil || !logFilterRegex.MatchString(output) {
 						fmt.Printf("%s\n", output)
 					}
 					out := fmt.Sprintf("%s\n", output)
 					if p.name == ptp4lProcessName {
 						if strings.Contains(output, ClockClassChangeIndicator) {
```

Comment:
Do we need this check anymore if we are doing clock class check  in interval ?

Reply:
I thought leaving it here for after ptp4l implements class change notification.
WDYT?